src/repos/LinkRepo.ts

```diff
@@ -0,0 +1,23 @@
+import { Prisma } from "@prisma/client";
+import { IAddLinkArgs, PromiseLink, PromiseLinkOrNull } from "@src/types";
+import { prisma } from "@src/util";
+
+/**
+ * Get first link.
+ */
+async function getFirst(args: Prisma.LinkFindFirstArgs): PromiseLinkOrNull {
```

Comment:
it's better to pass prisma into this getFirst, or to create a constructor for LinkRepo. 

Both ways will make it more testable. With current approach you need to mock ifile "@src/util".

Like this 

```suggestion
   getFirst(prisma: PrismaClient, args: Prisma.LinkFindFirstArgs)
```

Or like this 

```suggestion
 createLinkRepo(prisma: PrismaClient) {
  return {
  getFirst: (args: Prisma.LinkFindFirstArgs)
  }
 }
 ```
 
 Or you can do it with class, but i don't like classes in js. My prefered way is  the 1st suggestion.